### PR TITLE
.NET Interactive: Add_try_dotnet_to_batch_30

### PR DESCRIPTION
**Pull request by v-mepa**
fyi @mairaw 

## Add_try_dotnet_to_batch_30

S No. | URL | BounceRate | Try .NET Status | Comments
-- | -- | -- | -- | --
1 | https://docs.microsoft.com/en-us/dotnet/api/system.timespan.fromminutes | 9 | Batch 30 | Just first example / second throws exception
2 | https://docs.microsoft.com/en-us/dotnet/api/system.globalization.datetimeformatinfo.shortdatepattern | 8 | Batch 30 |  
3 | https://docs.microsoft.com/en-us/dotnet/api/system.enum.equals | 12 | Batch 30 |  
4 | https://docs.microsoft.com/en-us/dotnet/api/system.globalization.datetimeformatinfo.getalldatetimepatterns | 26 | Batch 30 | Except example 2 (something is wrong with this   example)
5 | https://docs.microsoft.com/en-us/dotnet/api/system.version.parse | 9 | Batch 30 |  
6 | https://docs.microsoft.com/en-us/dotnet/api/system.timespan.days | 11 | Batch 30 |  
7 | https://docs.microsoft.com/en-us/dotnet/api/system.type.basetype | 11 | Batch 30 | Only first example. Second example has env info.   Third is not runnable.
8 | https://docs.microsoft.com/en-us/dotnet/api/system.single.parse | 10 | Batch   30 | Except   example 2 (not runnable). Fix sample output of example 1.
9 | https://docs.microsoft.com/en-us/dotnet/api/system.decimal.divide | 13 | Batch   30 |  
10 | https://docs.microsoft.com/en-us/dotnet/api/system.decimal.compare | 11 | Batch   30 |  
11 | https://docs.microsoft.com/en-us/dotnet/api/system.nullable-1.value | 14 | Batch   30 |  
12 | https://docs.microsoft.com/en-us/dotnet/api/system.single.tryparse | 13 | Batch   30 | Fix   sample 1 output
13 | https://docs.microsoft.com/en-us/dotnet/api/system.bitconverter.tosingle | 11 | Batch   30 |  
14 | https://docs.microsoft.com/en-us/dotnet/api/system.timespan.milliseconds | 12 | Batch   30 |  
15 | https://docs.microsoft.com/en-us/dotnet/api/system.tuple-3 | 10 | Batch   30 |  
16 | https://docs.microsoft.com/en-us/dotnet/api/system.type.getnestedtypes | 11 | Batch   30 |  
17 | https://docs.microsoft.com/en-us/dotnet/api/system.icomparable-1.compareto | 14 | Batch   30 |  
18 | https://docs.microsoft.com/en-us/dotnet/api/system.array.rank | 9 | Batch   30 | Only   first sample. Second one is not runnable. Fix sample output
19 | https://docs.microsoft.com/en-us/dotnet/api/system.attribute.getcustomattributes | 26 | Batch   30 | Only   sample 2, 6, 8, 12. Fix sample output for these samples. others don't work on   Try .NET or environment info
20 | https://docs.microsoft.com/en-us/dotnet/api/system.type.tostring | 13 | Batch   30 | Only   sample 2. Other shows environment specific info.
21 | https://docs.microsoft.com/en-us/dotnet/api/system.timespan.duration | 18 | Batch   30 |

### CODE REVIEW

File: xml/System.Globalization/DateTimeFormatInfo.xml
Comment (mairaw) [inline xml/System.Globalization/DateTimeFormatInfo.xml:2920]:
```suggestion
 :::code language="vb" source="~/samples/snippets/visualbasic/VS_Snippets_CLR_System/system.Globalization.DateTimeFormatInfo.ShortDatePattern/VB/dtfi_shortdatepattern.vb" id="Snippet2":::
```

**Review by tarekgh — APPROVED** [review]

File: xml/System.Globalization/DateTimeFormatInfo.xml
Comment (mairaw) [inline xml/System.Globalization/DateTimeFormatInfo.xml:2920]:
```suggestion
 :::code language="vb" source="~/samples/snippets/visualbasic/VS_Snippets_CLR_System/system.Globalization.DateTimeFormatInfo.ShortDatePattern/VB/shortdatepattern1.vb" id="Snippet2":::
```

**Comment by mairaw** [comment]
@gewarren I hadn't validated the interactive samples were working yet. I'll review now, but on all the ones marked as .NET Interactive, please wait for my approval before merging.

**Comment by gewarren** [comment]
> @gewarren I hadn't validated the interactive samples were working yet. I'll review now, but on all the ones marked as .NET Interactive, please wait for my approval before merging.

Okay, will do. I assumed you had approved, sorry.

**Pull request merged**
